modules/smtp/modules.php

```diff
@@ -1573,7 +1574,7 @@ function rrmdir($dir) {
                     if (filetype($dir . "/" . $object) == "dir") {
                         rrmdir($dir . "/" . $object); 
                     } else {
-                        unlink($dir . "/" . $object);
+                        unlink($dir . "/" . $object);select_mailbox
```

Comment:
Probably accidentally pasted `select_mailbox` at the end of this line - it will generate a syntax error, can you remove please?